internal/objectstore/informer.go

```diff
@@ -71,6 +71,7 @@ func (f *informerFactory) ForResource(groupVersionKind schema.GroupVersionKind)
 	stopCh := f.informerContextCache.addChild(groupVersionKind)
 
 	gvr, _, err := f.client.Resource(groupVersionKind.GroupKind())
+	gvr.Version = groupVersionKind.Version
```

Comment:
This is actually the root cause of the error. With only this change on a cluster that doesn't support admissionregistration.k8s.io/v1 you wouldn't see a conversion error, but an empty listing.